androidHackathonApp/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: "com.android.application"
+apply from: rootProject.file("common.gradle")
+
+android {
+
+    defaultConfig {
+        applicationId "com.kcc.kmmhackathon.androidHackathonApp"
+    }
+}
+
+dependencies {
+
+    // Feature modules
+    implementation project(":lib-find-meal") // TODO: Rename to feature-find-meal
```

Comment:
This is my bad, I should have named it with `feature` not `lib` since it's a feature module